PluginsAndFeatures/azure-toolkit-for-intellij/resources/com/microsoft/azure/toolkit/operation/titles_account_intellij.properties

```diff
@@ -0,0 +1,21 @@
+account.sign_in=sign in to Azure
+account.sign_out=sign out from Azure
+account|auth_setting.load=load auth settings from file system
+account|auth_setting.persist=persist auth settings to file system
+account|auth_setting.update=update auth settings
+account|subscription.get_detail.selected=load details of all selected subscriptions
+account|subscription.get_detail=load details of subscription({0})
+account|subscription.get_tenant=get tenant id of subscription({0})
+account|subscription.list.tenant|authorized=load subscriptions from authorized tenants
+account|subscription.list.tenant=get all subscriptions of tenant({0})
+account|subscription.list.selected=list selected subscriptions
+account|subscription.load_all=load all available subscriptions from Azure
```

Comment:
how about replace "list.tenant" with "list_tenant" ? replace "list.selected" with "list_selected" as well?
Let "_" rather than "." as separator in action name.